Assets/Scripts/EnemySystem/ScriptableObjects/EnemyMovementStrategies/BabeMovementStrategy.cs

```diff
@@ -0,0 +1,188 @@
+﻿using EnemySystem.Monobehaviours;
+using UnityEngine;
+
+namespace EnemySystem.ScriptableObjects.EnemyMovementStrategies
+{
+    [CreateAssetMenu(fileName = "BabeMovementStrategy",
+        menuName = "ScriptableObjects/EnemyMovement/BabeMovement", order = 3)]
+    public class BabeMovementStrategy : EnemyMovementStrategy
+    {
+        #region Interface Variables
+    
+        //Atomic parameters of babe uwu
+        [SerializeField] private float chargeSpeed;
+        [SerializeField] private float jumpSpeedScale;
+        [SerializeField] private float miniJumpForce;
+        [SerializeField] private float superJumpForce;
+        [SerializeField] private float attackTime;
+        [SerializeField] private float waitTime;
+        [SerializeField] private int jumpRange;
```

Comment:
can this be a `float` to increase flexibility on the jump range?

Reply:
sure